app/controllers/reminders/reminder/add-reminder.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    addReminder() {
+      console.log('woot there it is');
```

Comment:
@martensonbj We're not using controllers. He was using our app to try and trouble shoot someone else's problem. We will delete the file because it's not in use.